tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+	"compilerOptions": {
+		"module": "ESNext",
+		"lib": ["ESNext"],
+		"skipLibCheck": true,
+		"allowImportingTsExtensions": true,
+		"allowSyntheticDefaultImports": true,
+		"isolatedModules": true,
+		"moduleDetection": "force",
+		"noEmit": true,
+		"baseUrl": ".",
+		"paths": {
+			"*": ["./node_modules/*"],
```

Comment:
won't this remap all imports to a subdir of node_modules?

Reply:
Good catch, previously had added this to the API config when I was still getting things running. Isn't needed now. Removed.